composer.json

```diff
@@ -8,7 +8,7 @@
     "require": {
         "drupal/entity_browser": "^2.5",
         "drupal/inline_entity_form": "^1.0-rc6",
-        "drupal/linkit": "^6.1.1",
+        "drupal/linkit": "^6.0.2",
```

Comment:
So may I suggest we try
```
~6.0.2 || ^6.1.2
```
This should
- [Disallow 6.0.1](https://jubianchi.github.io/semver-check/#/~6.0.2%20||%20^6.1.2/6.0.1)
- [Allow 6.0.2](https://jubianchi.github.io/semver-check/#/~6.0.2%20||%20^6.1.2/6.0.2)
- [Disallow 6.1.1](https://jubianchi.github.io/semver-check/#/~6.0.2%20||%20^6.1.2/6.1.1)
- [Allow 6.1.2](https://jubianchi.github.io/semver-check/#/~6.0.2%20||%20^6.1.2/6.1.2)
- [Allow 6.2.0](https://jubianchi.github.io/semver-check/#/~6.0.2%20||%20^6.1.2/6.2.0)

Reply:
Hi Andy, this is what I was thinking when restricting drupal/linkit to ^6.0.2:
- It covers anything above and including 6.0.2.
- It includes both 6.1.1 and 6.1.2.  The change from this pull request is compatible with 6.1.1 afterall.
- In the future it should also cover 6.2.x.

So perhaps ^6.0.2 is enough here?

Reply:
ok, I think it is 6.1.0 that is the problem, but then I don't think people would install that.

Reply:
> it is 6.1.0 that is the problem

That's a good point.  I missed 6.1.0.  Fixed now.